src/app/layout/Layout.tsx

```diff
@@ -14,14 +14,19 @@ export function Layout({ children }: { children: React.ReactNode }) {
     setSidebarOpen(!isSidebarOpen);
   };
 
+  const restrictPath = pathname === Routes.Playground || pathname === Routes.Login;
+
+
   return (
     <div className="flex">
-      {(pathname !== Routes.Login || isSidebarOpen) && (
+      {(!restrictPath || isSidebarOpen) && (
         <Sidebar isSidebarOpen={isSidebarOpen} toggleSidebar={toggleSidebar} />
```

Comment:
Let's move this check inside sidebar. Return null if it's a playground.

Reply:
fixed